src/test/java/hw2/Ex1/TestEx1.java

```diff
@@ -0,0 +1,105 @@
+package hw2.Ex1;
+
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import org.testng.annotations.Test;
+
+import java.util.*;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class TestEx1 {
+
+
+    @Test
+    public void ex1() {
+        WebDriverManager.chromedriver().setup();
+        WebDriver driver = new ChromeDriver();
+        driver.manage().window().maximize();
+
+
+        driver.get("https://epam.github.io/JDI/index.html");
+
+        // Assert Browser title
+        assertEquals(driver.getTitle().trim(), "Home Page");
+        // -----------------------------------------------------------
+
+        // Assert User name expected "PITER CHAILOVSKII
+        driver.findElement(By.id("user-icon")).click();
+        driver.findElement(By.cssSelector("#name")).sendKeys("epam");
+        driver.findElement(By.xpath("//input[@id='password']")).sendKeys("1234");
+        driver.findElement(By.id("login-button")).click();
+        WebElement loginedUserName = driver.findElement(By.id("user-name"));
+        assertEquals(loginedUserName.getText().trim(), "PITER CHAILOVSKII");
+        // -----------------------------------------------------------
+
+        // Assert 4 items on the header section "HOME", "CONTACT FORM", "SERVICE", "METALS & COLORS"
+        List<WebElement> elements = driver.findElements(By.tagName("li"));
+        List<String> foundedElements = new ArrayList<>();
+        List<String> expectedElements = new ArrayList<>(Arrays.asList("HOME", "CONTACT FORM", "SERVICE", "METALS & COLORS"));
+        for (int i = 0; i < elements.size(); i++) {
+            WebElement e = elements.get(i);
+            if (e.getText().equals("HOME") || e.getText().equals("CONTACT FORM") || e.getText().equals("SERVICE") || e.getText().equals("METALS & COLORS")) {
```

Comment:
it better use next construction for the comparing Strings with instances `"HOME".equals(e.getText())`. This allow avoid `NullPointerException`